h2o-py/h2o/model/dim_reduction.py

```diff
@@ -94,20 +93,23 @@ def proj_archetypes(self, test_data, reverse_transform=False):
         return h2o.get_frame(j["model_metrics"][0]["predictions"]["frame_id"]["name"])
 
 
-    def screeplot(self, type="barplot", server=False):
+    def screeplot(self, type="barplot", server=False, save_plot_path=None):
         """
         Produce the scree plot.
 
         Library ``matplotlib`` is required for this function.
 
         :param str type: either ``"barplot"`` or ``"lines"``.
         :param bool server: if true set server settings to matplotlib and do not show the graph
+        :param save_plot_path: a path to save the plot via using mathplotlib function savefig
+        
+        :returns: Object that contains the resulting scree plot (can be accessed like result.figure()).
         """
         # check for matplotlib. exit if absent.
         plt = get_matplotlib_pyplot(server)
         if plt is None:
-            return
-
+            return decorate_plot_result(figure=RAISE_ON_FIGURE_ACCESS)
```

Comment:
yes this is weird, noticed also before, but when I add it there, idea displays it in grey with "from h2o.plot import decorate_plot_result, get_matplotlib_pyplot, RAISE_ON_FIGURE_ACCESS" message and it builds successfully also without this import . But added in 1a66ba6

Reply:
oh, I see! It's due to the 
```python
from .metrics_base import *
```
this is not usually recommended if the module didn't define the `__all__` variable, as otherwise it will import everything in the module scope, including the variables/functions/classes/modules that were imported in that module…
Nevermind, you can leave the import, we can fix those `*` another time.